hera_pspec/pspecdata.py

```diff
@@ -4218,28 +4235,28 @@ def get_pspec_run_argparser():
     a = argparse.ArgumentParser(description="argument parser for pspecdata.pspec_run()")
 
     def list_of_int_tuples(v):
-        v = [tuple([int(_x) for _x in x.split()]) for x in v.split(",")]
+        v = [tuple([int(_x) for _x in x.split('~')]) for x in v.split(",")]
```

Comment:
this should definitely maintain backwards compatibility! this should be fairly easy by looking for the unique `~` character, which if it exists uses the new version, if not uses the old version. and a brief docstring on this function explaining the two expected formats would be nice.

Reply:
Agreed. I'll add an option for the old system if no "~" are registered.